skgstat/VariogramResult.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import pandas as pd
+from .Variogram import Variogram
+
+class LagClass(object):
+    def __init__(self, size):
+        self.size = size
+
+class VariogramResult(Variogram):
```

Comment:
Is it really a good idea to inherit from Variogram here? 
I think I don't really get why this is necessary.  I imagine, you can get somehow conused if a class has methods like fit and so on, that don't actually do anything. What do you think?

Reply:
Maybe yeah. Given the new describe(), maybe this class isn't that useful anymore even... It's not required by any of the rest of the code, and I guess I can reengineer the stuff I have that needs it...

Reply:
I would always go for dropping the class. On the other hand it does not harm the existing stuff. So afterall, it's kind of your decision, I guess

Reply:
Well, drop it then :)